### PR TITLE
Azure Function Apps Authentication Guide

### PR DESCRIPTION
**Pull request by smartzee797**

### CODE REVIEW

File: authentication-guides/azure-function-app/README.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
+---
+name: Azure Function Apps   # The name of the guide
+description: Connect Creator Studio to Azure Function Apps   # Brief description of the guide
+time_in_minutes: 30   # Time in minutes to complete the guide
+difficulty_level: Beginner  # Difficulty level: "Beginner", "Intermediate", or "Advanced"
```
Comment (amerchia-mw) [inline authentication-guides/azure-function-app/README.md:5]: This is definitely advanced

File: authentication-guides/azure-function-app/README.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
+---
+name: Azure Function Apps   # The name of the guide
+description: Connect Creator Studio to Azure Function Apps   # Brief description of the guide
+time_in_minutes: 30   # Time in minutes to complete the guide
```
Comment (amerchia-mw) [inline authentication-guides/azure-function-app/README.md:4]: I would quote this as 60 minutes to set up local Python and all the stuff around this

**Review by amerchia-mw — CHANGES_REQUESTED** [review]

File: authentication-guides/azure-function-app/README.md (hunk 2 above)
Comment (smartzee797) [reply authentication-guides/azure-function-app/README.md]: Done

File: authentication-guides/azure-function-app/README.md (hunk 1 above)
Comment (smartzee797) [reply authentication-guides/azure-function-app/README.md]: Done

**Review by amerchia-mw — APPROVED** [review]

**Pull request merged**